lib/logstash/outputs/mongodb.rb

```diff
@@ -34,17 +34,9 @@ class LogStash::Outputs::Mongodb < LogStash::Outputs::Base
   public
   def register
     require "mongo"
```

Comment:
how do you feel about moving the require up to the top? ala https://github.com/elastic/logstash/pull/3404#issuecomment-113170046